docs/apm/service-overview.asciidoc

```diff
@@ -14,17 +14,20 @@ high-level visibility into how a service is performing across your infrastructur
 
 [discrete]
 [[service-time-comparison]]
-=== Time series comparison
+=== Time series and expected bounds comparison
 
-Comparing how a service performs relative to a previous time frame can offer additional insight into
-the health of your services. For example, has latency been slowly increasing over time, or did the service
-experience a sudden spike--enabling a time series comparison can provide the answer.
+Comparing how a service performs relative to a previous time frame or to the 
+expected bounds from the corresponding {anomaly-job} can offer additional 
+insight into the health of your services. For example, has latency been slowly 
+increasing over time, did the service experience a sudden spike, is the actual 
```

Comment:
Is "actual" necessary? Makes me wonder "as opposed to what? predicted throughput?"
```suggestion
increasing over time, did the service experience a sudden spike, is the 
```